website/docs/docs/explore/access-dbt-insights.md

```diff
@@ -75,53 +75,53 @@ Let's use an example to illustrate how to run queries in <Constant name="query_p
-- If Kimiko wants to save the query, she can click **Bookmark** in the [query console menu](/docs/explore/navigate-dbt-insights#query-console-menu) to save it for future reference.
+- If you want to save the query, you can click **Bookmark** in the [query console menu](/docs/explore/navigate-dbt-insights#query-console-menu) to save it for future reference.
 
 :::tip Want to turn a query into a model?
-Coming soon &mdash; you'll be able to access the [<Constant name="cloud_ide" />](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) or [<Constant name="visual_editor" />](/docs/cloud/canvas) from the [Query console menu](/docs/explore/navigate-dbt-insights#query-console-menu) to promote your SQL into a reusable dbt model &mdash; all within <Constant name="cloud" />!
+You can access the [<Constant name="cloud_ide" />](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) or [<Constant name="visual_editor" />](/docs/cloud/canvas) from the [Query console menu](/docs/explore/navigate-dbt-insights#query-console-menu) to promote your SQL into a reusable dbt model &mdash; all within <Constant name="cloud" />!
 :::
 
 ### View results
 
-Using the same example, Kimiko can perform some exploratory data analysis by running the query and:
+Using the same example, you can perform some exploratory data analysis by running the query and:
 
 - Viewing results in **Results** tab &mdash; View the paginated results of the query.
 - Sorting results &mdash; Click on the column header to sort the results by that column.
 - Exporting to CSV &mdash; On the top right of the table, click the three-dot ellipsis (`...`) button and select **Export to CSV** to export the dataset.
-<Lightbox src="/img/docs/dbt-insights/insights-export-csv.jpg" width="95%" title="dbt Insights Export to CSV" />
+<Lightbox src="/img/docs/dbt-insights/insights-export-csv.png" width="95%" title="dbt Insights Export to CSV" />
 
 ### View details
-Kimiko can also view the details of the query by clicking on the **Details** tab:
+View the details of the query by clicking on the **Details** tab:
 - **Query metadata** &mdash; <Constant name="copilot" /> AI-generated title and description, the supplied SQL, and corresponding compiled SQL.
 - **Connection details** &mdash; Relevant data platform connection information.
 - **Query details** &mdash; Query duration, status, column count, row count.
 
-<Lightbox src="/img/docs/dbt-insights/insights-details.jpg" width="95%" title="dbt Insights Details tab" />
+<Lightbox src="/img/docs/dbt-insights/insights-details.png" width="95%" title="dbt Insights Details tab" />
 
 ### Chart results
 
-Kimiko can visualize the chart results of the query by clicking on the **Chart** tab to:
+Visualize the chart results of the query by clicking on the **Chart** tab to:
 - Select the chart type using the chart icon.
 - Choose from **line chart, bar chart, or scatterplot**.
 - Select the axis and columns to visualize using the **Chart settings** icon.
```

Comment:
can we update this screenshot as well? looks like this one got missed

Reply:
Yup! Got it